std/algorithm/comparison.d

```diff
@@ -1194,7 +1194,14 @@ levenshteinDistanceAndPath(alias equals = "a == b", Range1, Range2)
 
 // max
 /**
-Returns the maximum of the passed-in values.
+This function iterates the passed arguments and returns the maximum value.
+
+Params:
+    args = The values to select the maximum from. At least two arguments must
+    be passed.
+
+Returns: 
+    The maximum of the passed-in args.
```

Comment:
Probably should also explain what `MaxType!T` expands to, since it may not be obvious to an end user.